client/src/pages/news_notice/SelectedAnnouncement.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+
+import styled from "styled-components";
+import { parseError } from "../../graphql/client";
+import { useParams } from "react-router";
+
+import { useQuery } from "@apollo/client";
+
+import ReactHtmlParser, {
+  convertNodeToElement,
+  processNodes,
+} from "react-html-parser";
+
+import {
+  GET_ANNOUNCEMENT,
+  AnnouncementQueryData,
+  AnnouncementQueryVariable,
+} from "../../graphql/announcement/get_announcement.query";
+
+import { Announcement } from "../../types/announcement";
+import {
+  Typography,
+  withTheme,
+  Box,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Avatar,
+  Grid,
+  TablePagination,
+  TableFooter,
+} from "@material-ui/core";
```

Comment:
you are not even using most of these imported components.

and NEVER IMPORT LIKE
"@material-ui/core" -> this imports ALL COMPS into this comp and it increases load size. 

import it like; 

`import Box from '@material-ui/core/Box' <- this picks only one comp to be loaded into this